examples/esp-todomvc-react/js/todoItem.jsx

```diff
@@ -0,0 +1,111 @@
+/*global React */
+var app = app || {};
+
+(function () {
+    "use strict";
+
+    var ESCAPE_KEY = 27,
+        ENTER_KEY = 13;
+
+    app.TodoItem = React.createClass({
+
+        getInitialState: function () {
+            return {editText: ""};
+        },
+
+        handleDestroy: function () {
+            this.props.router.publishEvent("todoList", "todoDestroyed", { id: this.props.id });
```

Comment:
I wonder if we could clean things up by having `this.app.publish('event', {  })` to save duplicating topic everywhere?